packages/core/src/components/forms/_label.scss

```diff
@@ -8,19 +8,22 @@
 /*
 Labels
 
-Add labels to inputs to enhance the usability of your forms.
+Labels enhance the usability of your forms.
 
-Putting the `<input>` element _inside_ a `<label>` element also increases the area where the user
-can click to activate the control. Notice how in the example below, clicking a `<label>` below
-focuses its `<input>`.
+- Add extra information to the label with `span.pt-label-description`.
+
+- Putting the `<input>` element _inside_ a `<label>` element increases the area where the user
+can click to activate the control. Notice how in the examples below, clicking a `<label>` focuses its `<input>`.
```

Comment:
What about saying that `input` inside `label` is the "deprecated way" of buildings form, and mentioning that you won't be able to use for complex form field (i.e. control groups, numeric inputs...). In addition, we could say that this method of putting `input` inside `label` is an alternative way of binding label to field (`for` + `id` is the new way we recommend).